### DISCUSSION TITLE
No text formatting or images on built site

### DISCUSSION THREAD

**Discussion by wildetudor** [mmistakes/minimal-mistakes#2721] [Q&A] [answered]
Hello! I've forked https://academicpages.github.io/ into my own repo at https://github.com/wildetudor/tudorpopescu.github.io, replaced the content in /_pages/about.md. I built the site using Netlify (app.netlify.com), which connects to my GH repo.

However, the homepage (viewed in Edge/Chrome/Ffx) shows as unformatted text, and the avatar image also does not show:
![52eed311402bd0beb38ad2ca2001cc40bed3296d](https://user-images.githubusercontent.com/15835348/97625215-83f7c800-1a28-11eb-94e8-dab4f7a04086.jpeg)

I suspected some syntax error in the _config.yml or about.md (as in [this](https://github.com/academicpages/academicpages.github.io/issues/5) similar thread), but after checking all files in my repo, I"m at my wit's end what else to try! 

It might be telling that when I try to load my site from the tudorpopescu.github.io URL (instead of tudorpopescu.com, the URL defined in Netlify), I get a 404, whereas [other](https://nicofarr.github.io) websites based on the academicpages template load fine. 

Might this be a misconfiguration between GH and Netlify? If not easily fixed, might it be easier to use GitHub Pages instead?Unfortunately there is no more support from the academicpages repo itself, so am not sure where else to get help. Thanks in advance for any help!

**Comment by mmistakes** [comment]
Your paths are probably wrong by not setting `url` and/or `baseurl` properly in your _config.yml. 

You can verify this by viewing the source and checking the paths to images and the CSS in <head>. Neither are loading which is why you have missing images and no styling.

**Comment by iBug** [comment, marked as answer]
You can only use the domain `something.github.io` when `something` matches your username. See [About GitHub Pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites).

**Comment by wildetudor** [comment]
Thanks to both for replying. Indeed changing my repo name to match my GH username fixed the URL problem, and the formatting&images are displayed correctly. However, all browsers I tried to load the site in, display a warning icon about the site being "Not secure", which I've not seen for other websites build on the MM template:
![30-10-2020, 11 23 41](https://user-images.githubusercontent.com/15835348/97694279-9c0c2d80-1aa2-11eb-9266-52747b9b5151.jpg)
The Chrome [documentation](https://support.google.com/chrome/answer/95617?visit_id=637396501697839014-3205485761&p=ui_security_indicator&rd=1) suggests this means "something is severely wrong with the privacy of this site’s connection". How can I fix this privacy issue? Thanks again!

**Comment by iBug** [comment]
<https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/securing-your-github-pages-site-with-https>

**Comment by wildetudor** [comment]
Thanks! "Enforce HTTPS" was in fact already enabled:
![30-10-2020, 13 01 39](https://user-images.githubusercontent.com/15835348/97703217-8c93e100-1ab0-11eb-990e-2712da895af2.jpg)
The docs page mentions replacing all instances of HTTP with HTTPS, in case the site serves mixed content. In the HTML files under my _layouts folder I found no reference to HTTP however, and I cannot use search since my repo is forked.

I should add the "not secure" warning appears only when I navigate to the site using my custom URL (tudorpopescu.com), and not when I use the repo URL (https://wildetudor.github.io/). As I mentioned, my repo is linked to Netlify, whose settings don't seem to allow the necessary certificate in order to make the site use HTTPS. 
Thus, I again wonder if I can somehow remove my reliance on Netlify, since presumably I can use my custom URL in Github Pages alone, and would then presumably have no more problems with the HTTPS?!

**Comment by iBug** [comment]
Please refer to Netlify's docs on HTTPS: <https://docs.netlify.com/domains-https/https-ssl/#netlify-managed-certificates>

**Comment by wildetudor** [comment]
Thanks! I had actually followed that troubleshooter, but the error I get is not found there. Any thoughts about simply skipping Netlify altogether?

**Comment by iBug** [comment]
I recommend that you seek help on <https://talk.jekyllrb.com/> or <https://github.community/> rather than here. We want to focus the Issues board on this theme, not general support questions.

**Comment by wildetudor** [comment]
Of course, will do - thanks again for your help so far!
